src/main/java/com/hillel/items_exchange/dto/UserDto.java

```diff
@@ -19,23 +20,29 @@
 @Getter
 @Setter
 public class UserDto {
+    @ApiModelProperty(required = true)
     @NotEmpty(message = "{invalid.not-empty}")
     @Size(min = 2, max = 50, message = "{invalid.size}")
     private String username;
+    @ApiModelProperty(required = true)
     @NotEmpty(message = "{invalid.not-empty}")
-    @Size(max = 129, message = "{invalid.email}")
+    @Size(min = 1, max = 129, message = "{invalid.email}")
```

Comment:
Could you eplain why did you do that? We have annotation Email that works over standarts.

Reply:
Yes we have, but Swagger by default displays that this field can have min lenth = 0, so I added some constraint to show a requirement of email field